helm-charts/common/lvm-uservice/values.yaml

```diff
@@ -10,6 +10,9 @@ tgi:
   LLM_MODEL_ID: llava-hf/llava-v1.6-mistral-7b-hf
```

Comment:
could the models being used in the various stages be passed in as an environment variables to keep the Helm charts more generic, parameterized? Our course the infrastructure needs to scale to meet the needs of the various models.

Reply:
This came in from VisualQnA PR merge. I am unaware of what it is.

Reply:
Helm charts doesn't accept ENV variable input, this LLM_MODEL_ID can be overrided by --set like this:
helm install xxx xxx --set tgi.LLM_MODEL_ID="Your preferred model"